webinstall/github-eldev.bat fails to install Eldev properly

Hi,

The `webinstall/github-eldev.bat` fails to work as a GitHub workflow, although the `eldev.bat` is successfully installed, it fails to publish the location it is installed at.

To reproduce, create a github workflow file, that installs and runs `eldev.bat` on MS-Windows:

```yml
name: CI

on: [push, pull_request]

jobs:
  test:
    runs-on: ${{matrix.os}}

    strategy:
      matrix:
        os: [windows-latest]
        emacs_version: [ '28.1']

    steps:
    - name: Set up Emacs on Windows
      uses: jcs090218/setup-emacs-windows@master
      with:
        version: ${{matrix.emacs_version}}

    - name: Install Eldev on MS-Windows
      run: |
        curl.exe -fsSL https://raw.github.com/doublep/eldev/master/webinstall/github-eldev.bat | cmd /Q
 
    - name: Check out the source code
      uses: actions/checkout@v2

    - name: Test the action worked.
      run: |
        eldev help
``` 

It fails with:

```
...

2s
Run curl.exe -fsSL https://raw.github.com/doublep/eldev/master/webinstall/github-eldev.bat | cmd /Q
Microsoft Windows [Version 10.0.20348.707]
(c) Microsoft Corporation. All rights reserved.

D:\a\eldev\eldev>
10s
2m 7s
Run eldev help
eldev: D:\a\_temp\66bfe8f7-dc7c-4248-87dc-f22aaa1afcc2.ps1:2
Line |
   2 |  eldev help
     |  ~~~~~
     | The term 'eldev' is not recognized as a name of a cmdlet, function, script file, or executable
     | program. Check the spelling of the name, or if a path was included, verify that the path is correct
     | and try again.

Error: Process completed with exit code 1.
...
```

Although the tool is installed, it is not in the `PATH`. Not sure if this has ever worked before. PR to follow.

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -122,20 +122,29 @@ jobs:
           await io.rmRF(expandTilde("~/.local/bin"));
 
     # GitHub-specific installation specifically for workflows.
-    - name: Test GitHub-specific installation
-      uses: actions/github-script@v3
-      with:
-        script: |
-          const fs = require('fs');
-          const exec = require('util').promisify(require('child_process').exec);
-          const {getRawUrl,downloadScript,expandTilde} = require(`${process.env.GITHUB_WORKSPACE}/.github/workflows/util.js`);
-          if (fs.existsSync(expandTilde("~/.local/bin/eldev")))
-              throw("~/.local/bin/eldev already exists");
-          await io.mkdirP('tmp');
-          const script = await downloadScript(github, context, "webinstall/github-eldev", 'tmp');
-          const eldev_bin_url = getRawUrl(context,"bin/eldev");
-          await exec(`${script} ${eldev_bin_url}`);
-          const eldev_out = await exec(expandTilde("~/.local/bin/eldev"));
-          console.log(eldev_out.stdout);
-          await io.rmRF(expandTilde("~/.local/bin"));
+    - name: Test GitHub-specific installation (1/2)
+      if: "!startsWith (matrix.os, 'windows')"
+      run: |
+        if command -v eldev &> /dev/null
+        then
+         echo eldev is already installed.
+         exit 1
+        else
+          curl -fsSL https://raw.github.com/doublep/eldev/master/webinstall/github-eldev | sh
+        fi
 
+    - name: Test GitHub-specific installation on MS-Windows (1/2)
+      if: startsWith (matrix.os, 'windows')
+      run: |
+        if (Get-Command "eldev" -ErrorAction SilentlyContinue)
+        {
+          throw 'eldev is already installed.'
+        }
+        else
+        {
+          curl.exe -fsSL https://raw.github.com/doublep/eldev/master/webinstall/github-eldev.bat | cmd /Q
+        }
+
+    - name: Test GitHub-specific installation (2/2)
+      run: |
+        eldev help && rm ~/.local/bin/eldev*

--- a/webinstall/github-eldev.bat
+++ b/webinstall/github-eldev.bat
@@ -23,5 +23,5 @@ mkdir %ELDEV_BIN_DIR%
 
 curl.exe  -fsSL %URL% -o %ELDEV_BIN_DIR%\eldev.bat || exit /b
 
-echo %ELDEV_BIN_DIR% >> %GITHUB_PATH%
+echo %ELDEV_BIN_DIR%>> %GITHUB_PATH%
 

